unraid_mcp/core/utils.py

```
@@ -0,0 +1,68 @@
+    try:
+        value = float(int(bytes_value))
+    except (ValueError, TypeError):
+        return "N/A"
+    for unit in ["B", "KB", "MB", "GB", "TB", "PB"]:
+        if value < 1024.0:
+            return f"{value:.2f} {unit}"
+        value /= 1024.0
+    return f"{value:.2f} EB"
+
+
+def format_kb(k: Any) -> str:
```

Ruff (0.15.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `k`

(ANN401)